packages/Alert/src/Alert.tsx

```diff
@@ -1,178 +1,186 @@
+                    {type === "warning" && <WarningIcon />}
+                </>
+            )}
+        </div>
+    );
 };
 
 const renderDismissButton = (
-  ref: React.RefObject<HTMLDivElement>,
-  setShow: (show: boolean) => void,
-  onDismissClick?: () => void,
+    ref: React.RefObject<HTMLDivElement>,
+    setShow: (show: boolean) => void,
+    onDismissClick?: () => void
 ): JSX.Element => {
-  const action = (): void => {
-    if (onDismissClick) {
-      onDismissClick();
-    }
+    const action = (): void => {
+        if (onDismissClick) {
+            onDismissClick();
+        }
+
+        setShow(false);
+    };
 
-    setShow(false);
-  };
-
-  return (
-    <IconButton
-      appearance={{ type: 'ghost', variant: 'secondary' }}
-      className="ids-alert__dismiss-btn"
-      type="button"
-      size="medium"
-      icon={<Close size="medium" />}
-      onClick={action}
-    />
-  );
+    return (
+        <IconButton
+            appearance={{ type: "ghost", variant: "secondary" }}
+            className="ids-alert__dismiss-btn"
+            type="button"
+            size="medium"
+            icon={<Close size="medium" />}
+            onClick={action}
+        />
+    );
 };
 
 const renderAlertActionButton = (
-  style: Appearance,
-  button?: AlertButton,
+    style: Appearance,
+    button?: AlertButton
 ): JSX.Element => {
-  if (button == null || button.onClick == null || button.label == null) {
-    return <></>;
-  }
-
-  return (
-    <Button
-      appearance={style === 'horizontal' ? 'ghost' : 'secondary'}
-      size="small"
-      onClick={button.onClick}
-      className="ids-alert__action-btn"
-    >
-      {button.label}
-    </Button>
-  );
+    if (button == null || button.onClick == null || button.label == null) {
+        return <></>;
+    }
+
+    const ghostAppearance = getBrand() === "workleap" ? 
+        { type: "ghost", variant: "secondary" } as ButtonAppearance : 
+        "ghost";
+
+    return (
+        <Button
+            appearance={style === "horizontal" ? ghostAppearance : "secondary"}
+            size="small"
+            onClick={button.onClick}
+            className="ids-alert__action-btn"
+        >
+            {button.label}
+        </Button>
+    );
 };
 
 const Alert: React.FunctionComponent<AlertProps> = ({
-  title,
-  message,
-  metadata,
-  type,
-  appearance = 'card',
-  className,
-  closable = true,
-  icon,
-  onClose,
-  button,
-  dataTest,
-  ...rest
-}: AlertProps) => {
-  const classes = classNames(
-    'ids-alert',
-    `ids-alert--${appearance}`,
-    `ids-alert--${type}`,
+    title,
+    message,
+    metadata,
+    type,
+    appearance = "card",
     className,
-  );
-
-  const parentElement = React.useRef<HTMLDivElement>(null);
-  const [show, setShow] = React.useState(true);
-  const hasButton = button !== undefined;
-  const isHorizontal = appearance === 'horizontal';
-  const canBeClose = closable && !isHorizontal;
+    closable = true,
```

Comment:
I think a common prop name for this is dismissible

Reply:
true, but this was not changed and not part of the scope